great_expectations/execution_engine/sparkdf_execution_engine.py

```diff
@@ -3,7 +3,6 @@
 import copy
 import datetime
 import logging
```

Comment:
`sparkdf_execution_engine.py` is currently not being type-checked but it only has a few issues.
Can you remove it from the `pyproject.toml` file and deal with any issues?
Happy to help with any typing questions.

https://github.com/great-expectations/great_expectations/blob/develop/pyproject.toml#L63

Reply:
@Kilo59 Good idea -- done and committed -- please re-review!  Thanks!